src/internal/frame.hpp

```diff
@@ -7,6 +7,15 @@
 class Client;
 class Print;
 
+#if defined(ESP8266)
+# include <ESP8266WiFi.h>
+# include <ESPAsyncTCP.h>
+#endif
+#if defined(ESP32)
+# include <FreeRTOS.h>
+# include <AsyncTCP.h>
```

Comment:
This means, this library won't compile when `AsyncTCP` library is not installed.

Reply:
AsyncTCP is part of Espressif arduino-esp32 (https://github.com/espressif/arduino-esp32/), so it will be there. AsyncTCP is the base class used by ESPAsyncWebServer (https://github.com/me-no-dev/ESPAsyncWebServer) which is much faster / better than default espressif WebServer.

Reply:
Does the library makes sense to be compiled for ESP8266 at all? If not the code can be reduced to just 
# include <FreeRTOS.h>
# include <AsyncTCP.h>

I am using your library in a project that compiles on ESP8266 and EPS32, thats where I copied the include code from.